.env.example

```diff
@@ -0,0 +1,8 @@
+DATABASE_USERNAME=root
+DATABASE_PASSWORD=toor
+DATABASE_HOSTNAME=db
+
```

Comment:
thừa dòng này chị, dưới nữa

Reply:
để cách ra cho dễ nhìn ấy mà :v

Reply:
trong file này e thấy thg ko để trống dòng bh đâu

Reply:
cái này không ảnh hưởng vì file .env dòng trống bị bỏ qua.

These syntax rules apply to the .env file:

- Compose expects each line in an env file to be in VAR=VAL format.
- Lines beginning with # are processed as comments and ignored.
- Blank lines are ignored.
- There is no special handling of quotation marks. This means that they are part of the VAL.